iphone/metadata/pt-BR/release_notes.txt

```diff
@@ -1,3 +1,7 @@
-- iOS 18 Dark Mode: Introduced dark and tinted app icons.
-- Map Data: Updated as of October 1, now including isolines for Egypt, Adygea, Dagestan.
-- KML/KMZ/GPX: Fixed issues with importing files from different apps.
+• New OpenStreetMap data as of November 7
+• Improved iCloud Bookmarks and Tracks sync
+• Added water slides, carousels, ferris wheels, etc
+• Added traffic signals (more visible in car navigation)
+• Updated the OSM login screen
+• Fixed import of some ill-formed gpx/kml tracks
+• Various search improvements, translation updates & bug fixes
```

Comment:
• Novos dados do OpenStreetMap de 7 de novembro
• Melhoria na sincronização de favoritos e trilhas para o iCloud
• Adicionados toboáguas, carrosséis, rodas-gigantes, etc.
• Adicionados semáforos (mais visíveis na navegação de carro)
• Atualizada a tela de login do OSM
• Corrigida a importação de algumas trilhas gpx/kml malformadas
• Várias melhorias de pesquisa, atualizações de tradução e correções de bugs